velox/exec/Spiller.cpp

```diff
@@ -569,6 +569,24 @@ std::string Spiller::toString() const {
       spillFinalized_);
 }
 
+int32_t Spiller::Config::spillLevel(uint8_t startBitOffset) const {
+  VELOX_CHECK_LE(startBitOffset + hashBitRange.numBits(), 64);
+  const int32_t deltaBits = startBitOffset - hashBitRange.begin();
+  VELOX_CHECK_GE(deltaBits, 0);
+  VELOX_CHECK_EQ(deltaBits % hashBitRange.numBits(), 0);
```

Comment:
nit: perhaps, include deltaBits and hashBitRange.numBits() in the error message

nit: consider introducing a temp variable to avoid repeating hashBitRange.numBits()